Source/Fuse.Elements/Element.Validation.uno

```diff
@@ -45,9 +45,24 @@ namespace Fuse.Elements
 			if (ElementBatchEntry != null)
 				ElementBatchEntry.InvalidateRenderBounds();
 
+			if (!_hasNotifiedRenderBoundsChanged)
+			{
+				UpdateManager.AddDeferredAction(NotifyRenderBoundsChanged, UpdateStage.Layout, LayoutPriority.Post);
```

Comment:
Why is this dispatch done in the Layout stage and not just in the current stage?